### PR TITLE
Fix Name and firstname inverse

### PR DESCRIPTION
**Pull request by MarvinFlm**
# Description

Fix the name and the first name invers on the onboarding 

### Click Up reference

[[CU-XXXX](https://app.clickup.com/t/XXXX)](https://github.com/edgar-care/mobile/compare/CU-86c0nj1yq_Fix-Champs-inverss-dans-onboarding?expand=1)

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to a clickup task
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: patient_mobile/lib/screens/dashboard/accueil_page.dart [hunk 1]
```diff
@@ -123,8 +123,8 @@ class _HomePageState extends State<HomePage> {
               ),
               Text(
                 DateTime.now().hour > 18
-                    ? 'Bonsoir, ${infoMedical['firstname'] != null ? infoMedical["firstname"].toUpperCase() : 'medical Folder Doesnt work'} !'
-                    : 'Bonjour, ${infoMedical['firstname'] != null ? infoMedical["firstname"].toUpperCase() : 'medical Folder Doesnt work'} !',
+                    ? 'Bonsoir, ${infoMedical['name'] != null ? infoMedical["name"].toUpperCase() : 'medical Folder Doesnt work'} !'
+                    : 'Bonjour, ${infoMedical['name'] != null ? infoMedical["name"].toUpperCase() : 'medical Folder Doesnt work'} !',
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/accueil_page.dart:127]: Not the good one => `firstname`

File: patient_mobile/lib/screens/dashboard/chat_page.dart [hunk 2]
```diff
@@ -98,7 +96,7 @@ class _ChatPageState extends State<ChatPage> {
       return 'Dr. Edgard Test';
     }
 
-    return 'Dr. ${doctor1['firstname']} ${doctor1['name'].toUpperCase()}';
+    return 'Dr. ${doctor1['name']} ${doctor1['firstname'].toUpperCase()}';
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/chat_page.dart:99]: No need

File: patient_mobile/lib/screens/dashboard/chat_page.dart [hunk 3]
```diff
@@ -113,7 +111,7 @@ class _ChatPageState extends State<ChatPage> {
         doctorName.add('Dr. Edgard Test');
       } else {
         doctorName.add(
-            'Dr. ${doctor1['firstname']} ${doctor1['name'].toUpperCase()}');
+            'Dr. ${doctor1['name']} ${doctor1['firstname'].toUpperCase()}');
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/chat_page.dart:114]: No need

File: patient_mobile/lib/screens/dashboard/chat_page.dart [hunk 4]
```diff
@@ -83,8 +82,7 @@ class _ChatPageState extends State<ChatPage> {
         chatSelected = chat;
         doctorname = getDoctorName(chat);
       });
-    } else {
-    }
+    } else {}
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/chat_page.dart:85]: Really??

File: patient_mobile/lib/screens/2fa/account_page.dart [hunk 5]
```diff
@@ -182,7 +181,7 @@ class _AccountPageState extends State<AccountPage> {
                                             ),
                                             child: BoringAvatars(
                                               name:
-                                                  "${widget.infoMedical['firstname']} ${widget.infoMedical['name'].toUpperCase()}",
+                                                  "${widget.infoMedical['name']} ${widget.infoMedical['firstname'].toUpperCase()}",
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/2fa/account_page.dart:184]: No need

File: patient_mobile/lib/screens/dashboard/accueil_page.dart [hunk 6]
```diff
@@ -108,7 +108,7 @@ class _HomePageState extends State<HomePage> {
                       ),
                       child: BoringAvatars(
                         name:
-                            "${infoMedical['firstname'] ?? 'Ne fonctionne'} ${infoMedical['name'] != null ? infoMedical["name"].toUpperCase() : 'Pas'}",
+                            "${infoMedical['name'] ?? 'Ne fonctionne'} ${infoMedical['firstname'] != null ? infoMedical["firstname"].toUpperCase() : 'Pas'}",
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/accueil_page.dart:111]: No need

File: patient_mobile/lib/widget/navbarplus.dart [hunk 7]
```diff
@@ -207,7 +207,7 @@ class _NavbarPLusState extends State<NavbarPLus> {
                                               ),
                                               child: BoringAvatars(
                                                 name:
-                                                    "${infoMedical['firstname']} ${infoMedical['name'].toUpperCase()}",
+                                                    "${infoMedical['name']} ${infoMedical['firstname'].toUpperCase()}",
```
Comment (lucas-louis) [inline patient_mobile/lib/widget/navbarplus.dart:210]: No need

File: patient_mobile/lib/widget/navbarplus.dart [hunk 8]
```diff
@@ -221,7 +221,7 @@ class _NavbarPLusState extends State<NavbarPLus> {
                                                 CrossAxisAlignment.start,
                                             children: [
                                               Text(
-                                                "${infoMedical['firstname']} ${infoMedical['name'].toUpperCase()}",
+                                                "${infoMedical['name']} ${infoMedical['firstname'].toUpperCase()}",
```
Comment (lucas-louis) [inline patient_mobile/lib/widget/navbarplus.dart:224]: No need

**Review by lucas-louis — CHANGES_REQUESTED** [review]

File: patient_mobile/lib/screens/dashboard/accueil_page.dart [hunk 9]
```diff
@@ -123,8 +123,8 @@ class _HomePageState extends State<HomePage> {
               ),
               Text(
                 DateTime.now().hour > 18
-                    ? 'Bonsoir, ${infoMedical['firstname'] != null ? infoMedical["firstname"].toUpperCase() : 'medical Folder Doesnt work'} !'
-                    : 'Bonjour, ${infoMedical['firstname'] != null ? infoMedical["firstname"].toUpperCase() : 'medical Folder Doesnt work'} !',
+                    ? 'Bonsoir, ${infoMedical['firstname'] != null ? infoMedical["name"].toUpperCase() : 'medical Folder Doesnt work'} !'
+                    : 'Bonjour, ${infoMedical['firstname'] != null ? infoMedical["name"].toUpperCase() : 'medical Folder Doesnt work'} !',
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/accueil_page.dart:127]: No...

**Review by lucas-louis — CHANGES_REQUESTED** [review]

File: patient_mobile/lib/screens/dashboard/accueil_page.dart (hunk 6 above)
Comment (lucas-louis) [reply patient_mobile/lib/screens/dashboard/accueil_page.dart:111]: Still not

**Review by lucas-louis — CHANGES_REQUESTED** [review]

**Review by lucas-louis — APPROVED** [review]
LGTM

**Review by SpiritGY — APPROVED** [review]

**Pull request merged**